package.json

```diff
@@ -36,6 +36,8 @@
     "url": "https://github.com/react-native-async-storage/async-storage.git"
   },
   "scripts": {
+    "format:c": "clang-format -i $(git ls-files '*.cpp' '*.h' '*.m' '*.mm')",
```

Comment:
question: you have to have `clang-format` installed on the system in order for this to work - shall we put that info in  Contribution docs maybe?